### PR TITLE
Update Ride The Lightning to v0.14.1

### PR DESCRIPTION
**Pull request by kristapsk**
Update RTL to latest version v0.14.1.

https://github.com/Ride-The-Lightning/RTL/releases/tag/v0.14.1

### CODE REVIEW

**Comment by kristapsk** [comment]
@blckbx Could you check this with CLN? In release notes it mentions upgrade of c-lightning-REST too, but not sure is it mandatory or just for using P2TR as default onchain address type.

**Comment by blckbx** [comment]
> @blckbx Could you check this with CLN? In release notes it mentions upgrade of c-lightning-REST too, but not sure is it mandatory or just for using P2TR as default onchain address type.

Thx, I'll update CLN / RTL independently.

File: guide/lightning/web-app.md [hunk 1]
```diff
@@ -102,17 +102,16 @@ We do not want to run Ride the Lightning alongside bitcoind and lnd because of s
...
+  $ git verify-tag v0.14.1
+  > gpg: Signature made Sun Oct  8 00:25:22 2023 EEST
   > gpg:                using RSA key 3E9BD4436C288039CA827A9200C9E2BC2E45666F
-  > gpg: Good signature from "saubyk (added uid) <39208279+saubyk@users.noreply.github.com>" [unknown]
-  > gpg:                 aka "Suheb <39208279+saubyk@users.noreply.github.com>" [unknown]
-  > gpg: WARNING: This key is not certified with a trusted signature!
-  > gpg:          There is no indication that the signature belongs to the owner.
+  > gpg: Good signature from "saubyk (added uid) <39208279+saubyk@users.noreply.github.com>" [expired]
+  > gpg:                 aka "Suheb <39208279+saubyk@users.noreply.github.com>" [expired]
+  > gpg: Note: This key has expired!
```
Comment (blckbx) [inline guide/lightning/web-app.md:114]: Key is expired?

**Review by blckbx — COMMENTED** [review]
One nit re pgp key otherwise looks good to me.

File: guide/lightning/web-app.md (hunk 1 above)
Comment (kristapsk) [reply guide/lightning/web-app.md:114]: You had different output? Maybe I haven't refreshed keys.

Comment (blckbx) [reply guide/lightning/web-app.md:114]: Yeah, I ran across this when updating c-lightning-rest and renewed the key. Following the RTL guide from top to bottom, users will fetch the latest gpg keys anyway.

Comment (kristapsk) [reply guide/lightning/web-app.md:114]: Ok, will do it and update PR.

**Pull request merged**